services/blockchain-indexer/shared/dataService/business/pos/unlocks.js

```diff
@@ -36,16 +36,24 @@ const getPosUnlocks = async params => {
 		{ address: params.address },
 	);
 
+	const { height, genesis: { blockTime } } = await requestConnector('getNodeInfo', {});
+	const { header: { timestamp } } = await requestConnector('getBlockByHeight', { height });
```

Comment:
Can we use relevant methods from indexer here instead of directly invoking connector. 
Ref: `services/blockchain-indexer/shared/dataService/business/network.js`
Ref: `services/blockchain-indexer/shared/dataService/business/blocks.js`